src/servise/getClassForLink.ts

```diff
@@ -0,0 +1,9 @@
+import cn from 'classnames';
+
+export const getClassForLink = ({
+  isActive,
+}: {
+  isActive: boolean;
+}): string | undefined => {
```

Comment:
```suggestion
}): string => {
```

Why you need this?

Reply:
I just remember vscode write me about `| undefined`, maybe bug, because now its okey with only `string`